code_samples/simple_language/src/main/java/com/intellij/sdk/language/SimpleAnnotator.java

```diff
@@ -0,0 +1,59 @@
+package com.intellij.sdk.language;
+
+import com.intellij.lang.annotation.*;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.util.TextRange;
+import com.intellij.psi.*;
+import com.intellij.sdk.language.psi.SimpleProperty;
+import org.jetbrains.annotations.NotNull;
+import com.intellij.openapi.editor.DefaultLanguageHighlighterColors;
+
+import java.util.List;
+
+
+public class SimpleAnnotator implements Annotator {
+  // Define strings for the Simple language prefix - used for annotations, line markers, etc.
+  public static final String SIMPLE_PREFIX_STR = "simple";
+  public static final String SIMPLE_SEPARATOR_STR = ":";
+  
+  @Override
+  public void annotate(@NotNull final PsiElement element, @NotNull AnnotationHolder holder) {
+    // Ensure the Psi element is an expression
+    if ( !( element instanceof PsiLiteralExpression ) ) return;
+    
+    // Ensure the Psi element contains a string that starts with the key and separator
+    PsiLiteralExpression literalExpression = (PsiLiteralExpression) element;
+    String value = literalExpression.getValue() instanceof String ? (String) literalExpression.getValue() : null;
+    if ( ( value == null ) || !value.startsWith( SIMPLE_PREFIX_STR + SIMPLE_SEPARATOR_STR ) ) return;
```

Comment:
where do all these whitespaces around `(` and `)` come from? is this really the default formatting rules?